cmd/upgrade/roles/cmd.go

```diff
@@ -713,7 +713,8 @@ func upgradeOperatorPolicies(
 	case aws.ModeAuto:
 		if !confirm.Prompt(true, "Upgrade each operator role policy to latest version (%s)?", defaultPolicyVersion) {
 			if args.isInvokedFromClusterUpgrade {
-				return r.Reporter.Errorf("Operator roles need to be upgraded to proceed")
+				err := fmt.Errorf("Operator roles need to be upgraded to proceed")
+				return err
```

Comment:
```suggestion
				return fmt.Errorf("operator roles need to be upgraded to proceed")
```

Reply:
Update to just return `fmt.Errorf` 

Also its best practice not to return errors that start capitalized. This is because they can be wrapped and we can run into some funky formatting. Best to keep them lower case IMO.